metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -573,6 +573,17 @@ class MetalsLanguageServer(
     }
   }
 
+  @JsonNotification("metals/windowStateDidChange")
+  def windowStateDidChane(params: WindowStateDidChangeParams): Unit = {
+    CompletableFuture.completedFuture {
```

Comment:
Remove this line

Reply:
by the way, do we have `-Ywarn-value-discard` on? This would generate a warning

Reply:
we don't, that setting generates a lot of noise in my experience (especially in imperative code which is all over the place in metals).

Reply:
ah right, probably not the best idea on Metals (although Scala 2.13 will make the warning a bit nicer to deal with, allowing ascriptions to Unit to remove it)

Reply:
Done. Sorry.